dts/arm/st/l1/stm32l1.dtsi

```diff
@@ -32,6 +32,30 @@
 	};
 
 	soc {
+
+		flash: flash-controller@40023c00 {
+			compatible = "st,stm32-flash-controller", "st,stm32l1-flash-controller";
+			label = "FLASH_CTRL";
+			reg = <0x40023c00 0x400>;
+			interrupts = <4 0>;
+
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			flash0: flash@8000000 {
+				compatible = "soc-nv-flash";
+				label = "FLASH_STM32";
+
+				write-block-size = <4>;
+			};
+		};
+
+		rcc: rcc@40023800 {
+			compatible = "st,stm32-rcc";
+			#clock-cells = <2>;
+			reg = <0x40023800 0x400>;
+		};
```

Comment:
rcc/flash changes should go into its own commit. is this change made because others define flash/rcc at the top?

Reply:
@gmarull Yes. It is moved from the bottom to top.

Reply:
yes, this is disturbing to see a change on this part of the dts, could this be avoided ?

Reply:
@FRASTM This specific change was made to add nodes after rcc/flash, looks similar to other platform files.

Reply:
@sarthsmart can you move these changes to a separate commit?

Reply:
+1

Reply:
@FRASTM @erwango @gmarull Separate commit for changing the sequence of rcc/flash has been submitted.

Reply:
thanks, this PR could now be closed